modules/persistent-storage-csi-snapshots-provision.adoc

```diff
@@ -5,14 +5,14 @@
 [id="persistent-storage-csi-snapshots-provision_{context}"]
 = Provisioning a volume snapshot
 
-There are two ways to provision snapshots: xref:#snapshots-manual-provisioning[manually] and xref:#snapshots-dynamic-provisioning[dynamically].
-
-[id="snapshots-manual-provisioning"]
-== Manual provisioning
-
-As a cluster administrator, you can manually pre-provision a number of VolumeSnapshotContent objects. These carry the real volume snapshot details available to cluster users.
+There are two ways to provision snapshots: xref:#snapshots-dynamic-provisioning[dynamically] and xref:#snapshots-manual-provisioning[manually].
 
 [id="snapshots-dynamic-provisioning"]
```

Comment:
You need to include _context in all the anchor IDs in a module.

Reply:
Hm, feedback from @jboxman was that _context wasn't needed (https://github.com/openshift/openshift-docs/pull/19703/files#r391234058). Sounds like it's a rule, so I'll add `{context}`.